src/main/java/cc/polyfrost/oneconfig/internal/command/OneConfigCommand.java

```diff
@@ -32,104 +32,85 @@
 import cc.polyfrost.oneconfig.internal.gui.HudGui;
 import cc.polyfrost.oneconfig.libs.universal.ChatColor;
 import cc.polyfrost.oneconfig.libs.universal.UChat;
-import cc.polyfrost.oneconfig.utils.commands.annotations.*;
+import cc.polyfrost.oneconfig.utils.commands.annotations.Command;
+import cc.polyfrost.oneconfig.utils.commands.annotations.Descriptor;
+import cc.polyfrost.oneconfig.utils.commands.annotations.Greedy;
 import cc.polyfrost.oneconfig.utils.gui.GuiUtils;
 
 /**
  * The main OneConfig command.
  */
-@Command(value = "oneconfig", aliases = {"ocfg", "oneconfig"}, description = "Access the OneConfig GUI.")
+@Command(value = "oneconfig", aliases = {"ocfg"})
 public class OneConfigCommand {
 
-    @Main
-    private static void main() {
+    @Descriptor(value = "", description = "Opens the OneConfig GUI")
+    private void main() {
         GuiUtils.displayScreen(OneConfigGui.create());
     }
 
-    @SubCommand(value = "hud", description = "Open the OneConfig HUD config.")
-    private static class HUDSubCommand {
-        @Main
-        private static void main() {
-            GuiUtils.displayScreen(new HudGui());
-        }
+    @Descriptor(value = "", description = "Opens the OneConfig HUD configurator.", aliases = {"edithud"})
+    private void hud() {
+        GuiUtils.displayScreen(new HudGui());
     }
 
-    @SubCommand(value = "destroy", description = "Destroy the cached OneConfig GUI.")
-    private static class DestroySubCommand {
-        @Main
-        private static void main() {
-            OneConfigGui.INSTANCE = null;
-        }
+    @Descriptor(value = "", description = "Destroy the currently open OneConfig GUI.")
+    private void destroy() {
+        OneConfigGui.INSTANCE = null;
     }
 
-    @SubCommand(value = "profile", description = "Actions related to profiles.", aliases = {"profiles"})
-    private static class ProfileSubCommand {
-        @SubCommand(value = "list", description = "View all profiles", aliases = {"view"})
-        private static class List {
-            @Main
-            private static void main() {
-                StringBuilder builder = new StringBuilder()
-                        .append(ChatColor.GOLD).append("Available profiles:");
-                for (String profile : Profiles.getProfiles()) {
-                    builder.append("\n");
-                    if (OneConfigConfig.currentProfile.equals(profile)) builder.append(ChatColor.GREEN);
-                    else builder.append(ChatColor.RED);
-                    builder.append(profile);
-                }
-                UChat.chat(builder.toString());
+    @Command(value = "Profile", description = "Actions related to profiles.", aliases = {"profiles"})
```

Comment:
This could be something like @SubCommandGroup since command implies it is top level